packages/sign_in_with_apple/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.1.0
 
 - Expose `identityToken` to enable Firebase integration (https://github.com/aboutyou/dart_packages/issues/62)
+- Add support for passing a `nonce` to the authentication request
```

Comment:
@HenriBeck Extended the current CHANGELOG (and did not publish yet), as this should go together with the previous feature if I understood correctly, right?

Reply:
Yeah, both features are needed to allow integration with Firebase Auth.